### ISSUE TITLE
Add dream2nix to the flake registry

### ISSUE THREAD

**Issue by DavHau** [nix-community/dream2nix#378]
We should make a PR and add dream2nix to the flake registry.
That would allow users to run the dream2nix apps more easily.
We could name a few example commands commands to make a compelling argument.

**Comment by hsjobeki** [comment]
do you mean over here: https://github.com/NixOS/templates

**Comment by DavHau** [comment]
No, over here: https://github.com/NixOS/flake-registry/blob/master/flake-registry.json
